MN/Algoritmi/Algoritmi-ad-hoc/IsPositiveDefinite.m

```diff
@@ -0,0 +1,21 @@
+% checks if a matrix A is positive definite
+function is_pos_def = IsPositiveDefinite(A)
+	[n n] = size(A);
+
+	if isDiagonallyDominant(A) == 1
```

Comment:
A este poz definita daca:
1) A e diag dominanta
2) Aii > 0
3) A simetrica

```matlab
is_pos_def = 1;
% NISTE COMENTARII AICI
if isDiagonallyDominant(A) == 1 && diag(A) > 0 && A == A'
    return;
endif
```

fix sub asta vine celalalt for la care nu trebuie sa mai initializezi is_pos_def cu 1 ca este deja in caz ca nu s-a dat return.